### PR TITLE
MINOR: Add version check on enable-systest-events flag

### PR DESCRIPTION
**Pull request by granders**
Recent patch adding enable-systest-events flag without any version check breaks all uses of versioned console consumer. E.g. upgrade tests, compatibility tests etc.

Added a check to only apply the flag if running 0.10.0 or greater.

### CODE REVIEW

**Comment by granders** [comment]
@apovzner @ewencp
The issue was introduced in https://github.com/apache/kafka/pull/1278 (which I also reviewed...)

Unfortunately I didn't think of this in the recent code review, but I noticed this in a subsequent branch build. We'll see a large chunk of test failures without this present patch.

File: tests/kafkatest/services/console_consumer.py [hunk 1]
```diff
@@ -184,9 +183,12 @@ def start_cmd(self, node):
 
         # LoggingMessageFormatter was introduced after 0.9
         if node.version > LATEST_0_9:
-            cmd+=" --formatter kafka.tools.LoggingMessageFormatter"
+            cmd += " --formatter kafka.tools.LoggingMessageFormatter"
+
+        # enable systest events is only available in 0.10.0 and later
+        if node.version >= V_0_10_0_0:
```
Comment (ewencp) [inline tests/kafkatest/services/console_consumer.py]: I don't think it's an issue for this patch, but I'm realizing that we may not have clearly defined when we bump version numbers pre-release. Here, we're clearly saying that we've already marked `V_0_10_0_0`, but that's actually a bit weird given that we're still on a dev version, right? Or does this not pass until we actually remove the `dev0` suffix?

**Comment by ewencp** [comment]
@granders Good catch. Question about when exactly we do version bumping (and maybe about how exactly the comparison operator works?).
1. I assume this patch needs double commit as it is relevant to the 0.10 branch, but I want to have people explicitly request double commits. This is the case, correct?
2. Beyond the impact of this specific patch, what would have helped catch this? PR/branch builder? or does this require running against multiple branches to protect against? I want to figure out what the right process is so we catch these issues earlier.

File: tests/kafkatest/services/console_consumer.py (hunk 1 above)
Comment (granders) [inline tests/kafkatest/services/console_consumer.py]: `V_0_10_0_0` is defined independently of any kafkatest version bump. kafkatest version is the version of the kafkatest module, whereas an instance of `KafkaVersion` refers to a version of Kafka.

It's true that `KafkaVersion` inspects the **init** file to figure out what `TRUNK` should mean, but it strips any `dev*` suffix, so if we're in 0.10.0 branch, `TRUNK` actually equals `V_0_10_0_0`

**Comment by granders** [comment]
@ewencp Thanks for review! here are a few answers:

version bumping... I don't think it really matters in this case given how comparison works? (see my previous version-related comments)
1. Yes, we want double commit here
2. PR/branch builder would've caught this no matter irrespective of the branch. Even a run of just the sanity checks would have caught this particular issue.

Running sanity checks here:
https://jenkins.confluent.io/job/system-test-kafka-branch-builder/425/console

Full build here:
https://jenkins.confluent.io/job/system-test-kafka-branch-builder-2/54/console

**Comment by granders** [comment]
@ewencp Sanity checks now pass

**Comment by apovzner** [comment]
@granders thanks for fixing this.

**Comment by ewencp** [comment]
Full build had one failure, but its an existing transient issue. LGTM, thanks @granders!

**Pull request closed without merging**